modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/edit_kb_article.jsp

```diff
@@ -10,10 +10,13 @@
 <%
 EditKBArticleDisplayContext editKBArticleDisplayContext = new EditKBArticleDisplayContext(kbGroupServiceConfiguration, liferayPortletRequest, liferayPortletResponse, portletConfig);
 
+KBArticle kbArticle = editKBArticleDisplayContext.getKBArticle();
+
 if (editKBArticleDisplayContext.isPortletTitleBasedNavigation()) {
 	portletDisplay.setShowBackIcon(true);
 	portletDisplay.setURLBack(editKBArticleDisplayContext.getRedirect());
 
+	portletDisplay.setURLBackTitle(kbArticle.getTitle());
```

Comment:
You need to keep the three calls to `portletDisplay` methods as a block, and separate them from the `renderResponse` one. Something like:
```
portletDisplay.set...();
portletDisplay.set...();
portletDisplay.set...();l

renderResponse.set...();
```
Make sure this calls are alphabetically sorted.